tasks/common/launch.yml

```diff
@@ -5,7 +5,7 @@
 
 - name: Activate configured Systemd units
   become: true
-  when: unit_config is defined and unit_config|length > 0
+  when: unit_config is defined and unit_config|length > 0 and unit_item.type is not defined or unit_item.type != 'conf'
```

Comment:
may be worth clarifying the conditional after the `and` stmt -- e.g. `(unit_item.type is not defined or unit_item.type != 'conf')`?

Reply:
what exactly do you mean?

Reply:
Just that the `when` conditional as it exists now is a bit tough to read/follow. Adding some parentheses around the statement could help clarify the logic.

e.g. `when: unit_config is defined and unit_config|length > 0 and (unit_item.type is not defined or unit_item.type != 'conf')`

Reply:
Added suggested parentheses with ba7e772